传递token时插件报错

```bash
traefik-reverse-proxy-1  | time="2023-11-24T09:00:01Z" level=error msg="Recovered from panic in HTTP handler [192.168.31.60:49695 - /api]: plugins-local/src/github.com/SwanHubX/req-preprocess/preprocess.go:121:27: interface conversion: interface {} is nil, not float64" middlewareName=traefik-internal-recovery middlewareType=Recovery
traefik-reverse-proxy-1  | time="2023-11-24T09:00:01Z" level=error msg="Stack: goroutine 152 [running]:\ngithub.com/traefik/traefik/v2/pkg/middlewares/recovery.recoverFunc({0x5c5df48, 0xc0012d02a0}, 0xc000110f00)\n\tgithub.com/traefik/traefik/v2/pkg/middlewares/recovery/recovery.go:46 +0x219\npanic({0x4c36dc0?, 0xc000e6d428?})\n\truntime/panic.go:914 +0x21f\ngithub.com/traefik/yaegi/interp.runCfg.func1()\n\tgithub.com/traefik/yaegi@v0.15.1/interp/run.go:205 +0x185\npanic({0x4c36dc0?, 0xc000e6d428?})\n\truntime/panic.go:914 +0x21f\ngithub.com/traefik/yaegi/interp.runCfg.func1()\n\tgithub.com/traefik/yaegi@v0.15.1/interp/run.go:205 +0x185\npanic({0x4c36dc0?, 0xc000e6d428?})\n\truntime/panic.go:914 +0x21f\ngithub.com/traefik/yaegi/interp.typeAssert.func3(0xc00125f3f0)\n\tgithub.com/traefik/yaegi@v0.15.1/interp/run.go:455 +0x654\ngithub.com/traefik/yaegi/interp.runCfg(0xc000e18a00, 0xc00125f3f0, 0x0?, 0x4d74480?)\n\tgithub.com/traefik/yaegi@v0.15.1/interp/run.go:213 +0x285\ngithub.com/traefik/yaegi/interp.genFunctionWrapper.func1.1({0xc000e6ce28, 0x1, 0x1?})\n\tgithub.com/traefik/yaegi@v0.15.1/interp/run.go:1024 +0x7c5\nreflect.Value.call({0x40595e0?, 0xc00128fbf0?, 0x0?}, {0x4f10de4, 0x4}, {0xc000e6ce10, 0x1, 0x18?})\n\treflect/value.go:596 +0xce7\nreflect.Value.Call({0x40595e0?, 0xc00128fbf0?, 0x456869?}, {0xc000e6ce10?, 0x4ea72c0?, 0x1?})\n\treflect/value.go:380 +0xb9\ngithub.com/traefik/yaegi/interp.call.func9.2({0xc000e6ce10?, 0xc00128fbf0?, 0x0?})\n\tgithub.com/traefik/yaegi@v0.15.1/interp/run.go:1293 +0x36\ngithub.com/traefik/yaegi/interp.call.func9(0xc00125f130)\n\tgithub.com/traefik/yaegi@v0.15.1/interp/run.go:1313 +0xcc5\ngithub.com/traefik/yaegi/interp.runCfg(0xc000e0b680, 0xc00125f130, 0x6?, 0x4d74480?)\n\tgithub.com/traefik/yaegi@v0.15.1/interp/run.go:213 +0x285\ngithub.com/traefik/yaegi/interp.genFunctionWrapper.func1.1({0xc00128f980, 0x2, 0x4?})\n\tgithub.com/traefik/yaegi@v0.15.1/interp/run.go:1024 +0x7c5\ngithub.com/traefik/yaegi/stdlib._net_http_Handler.ServeHTTP(...)\n\tgithub.com/traefik/yaegi@v0.15.1/stdlib/go1_20_net_http.go:292\ngithub.com/traefik/traefik/v2/pkg/server/middleware.(*traceablePlugin).ServeHTTP(0x5c72820?, {0x5c5df48?, 0xc0012d02a0?}, 0x5c0af60?)\n\tgithub.com/traefik/traefik/v2/pkg/server/middleware/plugins.go:54 +0x26\ngithub.com/traefik/traefik/v2/pkg/middlewares/tracing.(*Wrapper).ServeHTTP(0xc00125c660, {0x5c5df48, 0xc0012d02a0}, 0xc000111000)\n\tgithub.com/traefik/traefik/v2/pkg/middlewares/tracing/wrapper.go:57 +0xf8\ngithub.com/traefik/traefik/v2/pkg/middlewares/accesslog.(*FieldHandler).ServeHTTP(0xc0006d3140, {0x5c5df48, 0xc0012d02a0}, 0x42d9380?)\n\tgithub.com/traefik/traefik/v2/pkg/middlewares/accesslog/field_middleware.go:31 +0x11e\ngithub.com/gorilla/mux.(*Router).ServeHTTP(0xc00088cde0, {0x5c5df48, 0xc0012d02a0}, 0xc000110f00)\n\tgithub.com/gorilla/mux@v1.8.0/mux.go:141 +0x22d\ngithub.com/traefik/traefik/v2/pkg/middlewares/recovery.(*recovery).ServeHTTP(0x30?, {0x5c5df48?, 0xc0012d02a0?}, 0x13?)\n\tgithub.com/traefik/traefik/v2/pkg/middlewares/recovery/recovery.go:32 +0x77\ngithub.com/traefik/traefik/v2/pkg/middlewares/accesslog.(*FieldHandler).ServeHTTP(0xc0006d3240, {0x5c5df48, 0xc0012d02a0}, 0x100?)\n\tgithub.com/traefik/traefik/v2/pkg/middlewares/accesslog/field_middleware.go:31 +0x11e\ngithub.com/traefik/traefik/v2/pkg/middlewares.(*HTTPHandlerSwitcher).ServeHTTP(0x4132e5?, {0x5c5df48, 0xc0012d02a0}, 0x5c0ca01?)\n\tgithub.com/traefik/traefik/v2/pkg/middlewares/handler_switcher.go:23 +0x59\ngithub.com/traefik/traefik/v2/pkg/middlewares/requestdecorator.(*RequestDecorator).ServeHTTP(0xc0006ca4f8, {0x5c5df48, 0xc0012d02a0}, 0xc000110e00, 0xc00086d7e0)\n\tgithub.com/traefik/traefik/v2/pkg/middlewares/requestdecorator/request_decorator.go:47 +0x319\ngithub.com/traefik/traefik/v2/pkg/server.createHTTPServer.WrapHandler.func3.1({0x5c5df48?, 0xc0012d02a0?}, 0xc00128f860?)\n\tgithub.com/traefik/traefik/v2/pkg/middlewares/requestdecorator/request_decorator.go:89 +0x65\nnet/http.HandlerFunc.ServeHTTP(0xc00090d3b0?, {0x5c5df48?, 0xc0012d02a0?}, 0x9?)\n\tnet/http/server.go:2136 +0x29\ngithub.com/traefik/traefik/v2/pkg/middlewares/forwardedheaders.(*XForwarded).ServeHTTP(0xc00090d3b0, {0x5c5df48, 0xc0012d02a0}, 0xc000110e00)\n\tgithub.com/traefik/traefik/v2/pkg/middlewares/forwardedheaders/forwarded_header.go:192 +0xbd\ngithub.com/traefik/traefik/v2/pkg/server.createHTTPServer.AllowQuerySemicolons.func5({0x5c5df48, 0xc0012d02a0}, 0xc000110e00)\n\tnet/http/server.go:2960 +0x1e2\nnet/http.HandlerFunc.ServeHTTP(0xc00127d098?, {0x5c5df48?, 0xc0012d02a0?}, 0xc00086d958?)\n\tnet/http/server.go:2136 +0x29\ngolang.org/x/net/http2/h2c.h2cHandler.ServeHTTP({{0x5c16860?, 0xc000d110f8?}, 0xc00090d400?}, {0x5c5df48, 0xc0012d02a0}, 0xc000110e00)\n\tgolang.org/x/net@v0.17.0/http2/h2c/h2c.go:125 +0x68e\nnet/http.serverHandler.ServeHTTP({0xc00128e1e0?}, {0x5c5df48?, 0xc0012d02a0?}, 0x6?)\n\tnet/http/server.go:2938 +0x8e\nnet/http.(*conn).serve(0xc000902120, {0x5c72820, 0xc000d04090})\n\tnet/http/server.go:2009 +0x5f4\ncreated by net/http.(*Server).Serve in goroutine 98\n\tnet/http/server.go:3086 +0x5cb\n" middlewareName=traefik-internal-recovery middlewareType=Recovery
```
token 为：
```
eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImlhdCI6MTUxNjIzOTAyMn0.NHVaYe26MbtOYhSKkoKYdFVomg4i8ZJd8_-RU8VNbftc4TSMb4bXP3l3YlNWACwyXPGffz5aXHc6lty1Y2t4SWRqGteragsVdZufDn5BlnJl9pdR_kdVFUsra2rWKEofkZeIC4yWytE58sMIihvo9H1ScmmVwBcQP6XETqYd0aSHp1gOa9RdUPDvoXQ5oqygTqVtxaDr6wUFKrKItgBMzWIdNZ6y7O9E0DhEPTbE9rfBo6KTFsHAZnMg4k68CDp2woYIaXbmYTWcvbzIuHO7_37GT79XdIwkm95QJ7hYC9RiwrV7mesbY4PAahERJawntho0my942XheVLmGwLMBkQ
```
Payload ：
```json
{
  "sub": "1234567890",
  "name": "John Doe",
  "admin": true,
  "iat": 1516239022
}
```

--- a/preprocess.go
+++ b/preprocess.go
@@ -114,7 +114,7 @@ func (p *Preprocess) parseJWT(req *http.Request) {
 		}
 		// 3. 验证jwt凭证是否过期
 		var v map[string]interface{}
-		if err = json.Unmarshal(payload, &v); err != nil {
+		if err = json.Unmarshal(payload, &v); err != nil || v["exp"] == nil {
 			return
 		}
 		exp := int64(math.Floor(v["exp"].(float64)))

